main/app.go

```diff
@@ -18,31 +21,46 @@ type App struct {
 	DB     *sql.DB
 }
 
-//Initialize method
-func (a *App) Initialize(user, password, dbname string) {
-	connectionString := fmt.Sprintf("user=%s password=%s dbname=%s sslmode=disable", user, password, dbname)
+//Initialize func: init server according configuration structure
+func (a *App) Initialize(config Configuration) {
 	var err error
-	a.DB, err = sql.Open("postgres", connectionString)
-	if err != nil {
-		log.Fatal(err)
+	if !config.Debug {
```

Comment:
Maybe move two cases to two functions?

Reply:
maybe, will think about it..

Reply:
so i doesn't like how it looks when i split this to two func